app/services/requirements_from_xlsx_seeder.rb

```diff
@@ -0,0 +1,132 @@
+class RequirementsFromXlsxSeeder
+  def self.seed
+    file_name = "#{Rails.root}/db/templates/Core Permit Requirements List.xlsx"
```

Comment:
can be changed later, but `file_name` can probably be accepted as a param to the static method with the default param as the value